Make.rules

```diff
@@ -36,19 +36,27 @@
 
 .SECONDARY:
 
+CHAR_SUBSYSTEM=$(subst 0x,\x,$(SUBSYSTEM))
+
 ifeq ($(IS_MINGW32),)
 %.efi: %.so
 	@$(ECHO) "  OBJCOPY  $(notdir $@)"
 	$(HIDE)$(OBJCOPY) -j .text -j .sdata -j .data -j .dynamic -j .rodata -j .rel \
 		    -j .rela -j .rel.* -j .rela.* -j .rel* -j .rela* \
 		    -j .areloc -j .reloc $(FORMAT) $*.so $@
+ifeq ($(ARCH),riscv64)
```

Comment:
This could do with being a tool e.g. installed script
It'll break things that don't use generator mode (e.g.fwupd-efi)

Reply:
I'm not sure I understand what you suggest by "installed script". And I'm not sure how much we want to go the extra mile to cater for downstream projects, when our job is to produce working `.efi` executables by whatever means necessary, which is what downstream should expect, and which is exactly what we are doing here.

Now I am open to suggestions that can help dowstream, but you will have to be very explicit about what you want here, as I am not planning to be looking at fwupd-efi or other downstream projects myself, to figure out how fixing our mandate is going to conflict with stuff they might be relying on... So if you could elaborate on how this could be altered, I would appreciate it.

Reply:
> I'm not sure I understand what you 

Having a rethink :)
TLDR: Don't worry


fwupd-efi doesn't need it, they run generator mode as I wrote the code for that. Really, I wished everyone would copy my idea from there as then crt0 can be altered easily without us being worried about breakages. 

Shim is moving to objcopy mode which won't need it either
*this may also make this complicated later but hey ho, I have an idea

fwupd-efi is the only current *live* downstream for RISCV64 (which as I said is unaffected per my code)

Future downstreams will need to make changes anyways for new platforms so they can adapt to this